packages/nns/README.md

```diff
@@ -420,6 +421,17 @@ it is fetched using a query call.
 | ------------------ | ------------------------------------------------- |
 | `listKnownNeurons` | `(certified?: boolean) => Promise<KnownNeuron[]>` |
 
+##### :gear: lastestRewardEvent
+
+Returns the latest reward event.
+
+If `certified` is true, the request is fetched as an update call, otherwise
+it's fetched using a query call.
+
+| Method               | Type                                            |
+| -------------------- | ----------------------------------------------- |
+| `lastestRewardEvent` | `(certified?: boolean) => Promise<RewardEvent>` |
```

Comment:
Shouldn't it be called get* or load* or query* or something?

Reply:
yes, good point. The endpoint is called `get_latest_reward_event`. I'll keep the same name